ls1intum/Artemis
# Feature Guided Tour: Tutor assessment tutorial

### Checklist

### Motivation and Context
This feature introduces a guided tutorial for the tutor assessment process:
- [x] Add tour steps and translations
- [x] Enable navigation through multiple component pages
- [x] Start from the right tour step based on the current component page
- [x] Enable assessment of text example submissions
- [x] Restart tutorial by deleting `tutor_participations`

### Preparation Steps for Assessment Tour for PROD
1. Update `application-prod.yml` file on the server and add configuration for `tutor_assessment_tour`
2. Create text exercise with the title `Patterns in Software Engineering` in the `Tutorial Course`
3. Create example submission for review (1x) and for assessment (1x)
4. Start exercise and submit a solution

### Steps for Testing
1. Go to Course Administration and start or restart the tutor assessment tour
2. Go through the tutorial steps
3. When restarting the tutorial at the end, the tutor participation should be removed for the example review and example assessment

### Screenshots
![video](https://user-images.githubusercontent.com/33764166/74788597-81967900-52b2-11ea-9c5e-d4f34036fa4e.gif)
